tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   parameters:
     AgentPool : 'Linux-CPU'
     JobName: 'Linux_CI_Dev'
-    BuildCommand: 'tools/ci_build/github/linux/run_dockerbuild.sh -o ubuntu16.04 -d cpu -r $(Build.BinariesDirectory) -x "--use_mklml --use_llvm --use_nuphar --use_dnnl --use_tvm --use_featurizers --build_wheel --build_java --enable_language_interop_ops"'
```

Comment:
Why do you remove it?

Reply:
This is only for validating the pull requests. Not used in any packaging pipelines.

Reply:
Sounds good. I have restored featurizers.